src/main/java/hudson/plugins/scm/koji/model/RPM.java

```diff
@@ -86,6 +92,29 @@ public boolean hasUrl() {
         return url != null;
     }
 
+    public void setHashSum(File file) throws IOException, NoSuchAlgorithmException {
+        if (file == null) {
+            hashSum = null;
+            return;
+        }
+        byte[] buffer = new byte[(int) file.length()];
+        FileInputStream stream = new FileInputStream(file);
```

Comment:
- use try with resources for this input stream
 - Files#newInputStream is more robust solution https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#newInputStream-java.nio.file.Path-java.nio.file.OpenOption...-
 - you should use interface when possible -> `InputStream is = ...`